src/components/SingleTeam.jsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+const SingleTeam = ({ title, job, photo }) => {
+    return (
+        <>
+            {/* Single Card */}
+
+            <div className=' w-[176px] rounded-[25px] max-w-sm  bg-light-cyan'>
+                <Image src={photo} alt='My Image' width={176} height={190} />
+                <div className='bg-black w-full h-[5px] mt-0 bg-phosphorescent '></div>
+                <div className='p-5'>
+                    <h5 className='mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white pb-[33px]'>
```

Comment:
You need to refer to theme (tailwind.config.js)
1. font-bold is not a choice in our theme mentioned above, we have only (light, normal, medium, semibold). but if you want to write another value that doesnt exit there, you should use [ ], to be `font-[bold]`.
2. use ONLY colors in our theme, you can replace `text-gray-900` to `text-gray`, since our gray color in our theme is a specific #value.
3. Remove unused `dark:text-white`, since we do not intend to make a dark/light mode in the short term.